src/transport.ts

```diff
@@ -0,0 +1,46 @@
+import { WebRTCDialOptions }  from './options';
+//import { fromMultiAddr }      from './sdp'
+import { Connection }         from '@libp2p/interface-connection';
+import {CreateListenerOptions}from '@libp2p/interface-transport'
+import {Listener, Transport } from '@libp2p/interface-transport'
+import {DialOptions, symbol } from '@libp2p/interface-transport'
+import { logger }             from '@libp2p/logger'
+import { Multiaddr }          from '@multiformats/multiaddr';
+import { v4 as genUuid }      from 'uuid';
+
+const log = logger('libp2p:webrtc:transport')
+
+export class WebRTCTransport implements Transport {
+
```

Comment:
Mix of tabs and spaces in diff files.  We should find out Protocol's preferred tab/space.

Reply:
I could probably borrow this:

https://github.com/protocol/arg/blob/master/.prettierrc